app/editor/src/components/form/radio/RadioGroup.tsx

```diff
@@ -83,30 +93,36 @@ export const RadioGroup = <OT extends string | number | IOptionItem | HTMLOption
   };
 
   return (
-    <>
-      {label && (
-        <Row>
-          <label htmlFor={`dpn-${name}`} data-for="main-tooltip" data-tip={tooltip}>
-            <b>{label}</b>
+    <styled.RadioGroup
+      name={name}
+      className={`frm-in rag${className ? ` ${className}` : ''}`}
+      {...rest}
+    >
+      <div>
+        {label && (
+          <label
+            data-for="main-tooltip"
+            data-tip={tooltip}
+            className={rest.required ? 'required' : ''}
+          >
+            {label}
           </label>
-        </Row>
-      )}
-      <styled.Div direction={direction} className="frm-in rag">
+        )}
         {options
           ? options.map((option) => {
```

Comment:
I think this component needs to be refactored at some point.  Has to be a way to remove the duplication.